config/config.go

```diff
@@ -29,6 +30,9 @@ func InitConfig() {
 	RelayUrl = os.Getenv("RELAY_URL")
 	MemeUrl = os.Getenv("MEME_URL")
 	RelayAuthKey = os.Getenv("RELAY_AUTH_KEY")
+	AdminStrings := os.Getenv("SUPER_ADMINS")
```

Comment:
@elraphty can you change this value to just `ADMINS` aswell `SUPER_ADMINS` seems a bit much especially since we don't even have the concept of admins implemented yet